src/components/TimeCardPage/CellTypes/CommentModals/ShowReportModal.tsx

```diff
@@ -110,27 +114,60 @@ export default function ShowReportModal({
           <ModalBody>
             {reports.map(
               (report) => (
-                <HStack>
+                <VStack spacing={4} align="stretch">
                   {/* TODO: add UserDisplay card once pr merged in*/}
-                  <Editable
+                  <FormControl>
+                    <FormLabel>
+                      Reason for report:
+                    </FormLabel>
+                    <Editable
+                      isDisabled={!isEditable}
+                      defaultValue={report.Content}
+                      onSubmit={(value) => saveEditedReport(setReports, reports, report, createNewReport(user, value as ReportOptions, report.Notified, report.Explanation))}
```

Comment:
Can we separate these function calls out into two lines (`saveEditedReport` and `createNewReport`), just for readability